### ISSUE TITLE
cdn prefix and versioning (digest)

### ISSUE THREAD

**Issue by Enalmada** [vmunier/play-scalajs-scripts#6]
This project made it so easy to get a sample up and running with scalajs and I think everyone could keep using it in production if it supported cdn and versioning.     

For cdn, I think passing an optional cdn prefix to the scripts would work no problem: 
```
@playscalajs.html.scripts("client", cdn="http://cdn.domain.com")
```  

For versioning, play recommends you use routes.Assets.versioned. This is easy in a normal project but I tried and am not sure how to get it working in a subproject like this.  I think versioned needs a routes file hook and not sure how to work in a routes file:
```
<script src="@cdn@routes.Assets.versioned{s"$assetsPath/$jsdeps"}" type="text/javascript"></script>
```

Any chance you could get versioning working?  I think everyone that ever wants to run this with a cdn would be very grateful.  Thank you for such a nice community project.

**Comment by vmunier** [comment]
Hi @Enalmada,
I've just published play-scalajs-scripts `v0.4.0`, which introduces an `assets` parameter (String=>String).
Here's how you can now use routes.Assets.versioned along with a CDN:
```
@playscalajs.html.scripts(projectName = "client", fileName => cdn + routes.Assets.versioned(fileName).toString)
```

Let me know if that works for you!

**Comment by Enalmada** [comment]
Works perfectly!  Thank you very much for your help with this.
